.github/workflows/deploy-docs.yml

```diff
@@ -0,0 +1,38 @@
+name: Build and Deploy docs
+
+on: [push, pull_request]
+
+jobs:
+  build-docs:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Conda
+      uses: s-weigand/setup-conda@v1
+      with:
+        activate-conda: false
+        conda-channels: conda-forge
+
+    - name: Build environment
+      shell: bash -l {0}
+      run: |
+         conda create --name OHW-site --file .binder/conda-linux-64.lock
```

Comment:
This is probably not needed here. We can copy the env creation lines from Travis-CI.

Reply:
You'd think it would explode if it didn't find that file.